packages/gateway/src/gateway.js

```diff
@@ -193,6 +193,14 @@ async function gatewayFetch(
       signal: controller.signal,
       headers: getHeaders(request),
     })
+  } catch (error) {
+    if (controller.signal.aborted) {
+      return {
+        url: gwUrl,
+        aborted: true,
```

Comment:
Can we please be specific that this is a timeout (`timeout: true`), or have `aborted: reason` - where reason is timeout.

We could maybe roll `requestPreventedCode` into this:

```js
{
  url: gwUrl,
  aborted: true,
  reason: REQUEST_PREVENTED_RATE_LIMIT_CODE
}
```

And for timeout:

```js
{
  url: gwUrl,
  aborted: true,
  reason: 'TIMEOUT'
}
```

...for all other successful you set `aborted: false`.

Just trying to get a standard return type out of this.